l1-contracts/test/fee_portal/depositToAztecPublic.t.sol

```diff
@@ -64,29 +65,13 @@ contract DepositToAztecPublic is Test {
     feeJuicePortal.depositToAztecPublic(bytes32(0x0), 1, bytes32(0x0));
   }
 
-  function test_GivenSufficientBalance(uint256 _numberOfRollups) external {
+  function test_GivenSufficientBalance() external {
     // it should create a message for the newest version
```

Comment:
This no longer deals with newest version per say

Reply:
💀 never writing comments again, will only come back to hount me. 

Practically here though it is still the newest and it would fail if not due to the inbox being pulled based on what is the canonical in the registry - but I get your point 😆